charts/argo-workflows/Chart.yaml

```diff
@@ -16,3 +16,4 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - "[Added]: Add controller namespaceParallelism param to configmap"
+    - "[Added]: Add controller initialDelay param to configmap"
```

Comment:
The changes here should describe the current updates not cumulative ones.
So they should replace previous changes, not add to them.

Reply:
Thanks @vladlosev.  I just removed the redundant changelog entry.

Reply:
@vladlosev, let me know if you'd like me to bump version again to pass lint check or if you'll do so before merging.  Seems it slipped in the time passed.

Reply:
Yes, please do.

Reply:
Thanks.  Ticked up chart version, `chart-test` workflow can now be rerun.